test/AWS.SessionProvider.Test.csproj

```diff
@@ -33,14 +40,17 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="AWSSDK.Core, Version=3.3.0.0, Culture=neutral, PublicKeyToken=885c28607f98e604, processorArchitecture=MSIL">
-      <HintPath>..\packages\AWSSDK.Core.3.3.8\lib\net45\AWSSDK.Core.dll</HintPath>
-      <Private>True</Private>
+      <HintPath>..\packages\AWSSDK.Core.3.3.101.8\lib\net45\AWSSDK.Core.dll</HintPath>
     </Reference>
     <Reference Include="AWSSDK.DynamoDBv2, Version=3.3.0.0, Culture=neutral, PublicKeyToken=885c28607f98e604, processorArchitecture=MSIL">
-      <HintPath>..\packages\AWSSDK.DynamoDBv2.3.3.4\lib\net45\AWSSDK.DynamoDBv2.dll</HintPath>
-      <Private>True</Private>
+      <HintPath>..\packages\AWSSDK.DynamoDBv2.3.3.101.7\lib\net45\AWSSDK.DynamoDBv2.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.VisualStudio.TestPlatform.TestFramework, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Why aren't all these dll just <PackageReference ?

Reply:
See: https://github.com/aws/aws-aspnet-cognito-identity-provider/blob/master/test/unit/Amazon.AspNetCore.Identity.Cognito.Tests/Amazon.AspNetCore.Identity.Cognito.Tests.csproj

Reply:
It may be due to the csproj format, but I don't know for certain.

Reply:
If it were .NET Core it would be a simpler format.  I used the Visual Studio UI and that's how it updated the csproj file.

Reply:
I'll update to the latest. Stand by for new commit to PR.

Reply:
+1 on the old csproj format. It will be cleaner once we target the net standard .